docs/editing.md

```diff
@@ -216,3 +216,39 @@ Mark yourself as OoO in one of the reviews you're editing in the [joss-reviews](
 Ooo bot will then respond to any mentions in the [joss-reviews](https://github.com/openjournals/joss-reviews) repository to let people know you're away.
 
 **Note, if you're planning on being out of the office for more than two weeks, please let the JOSS editorial team know.**
+
+## Managing notifications
+
+Being on the JOSS editorial team means that there can be a _lot_ of notifications from GitHub if you don't take some proactive steps to minimize noise from the reviews repository.
+
+### Things you should do when joining the editorial team
+
+**Unsubscribe from the reviews repository on GitHub**
+
+When you're added to the editorial team on GitHub, you will almost certainly find yourself subscribed (watching) to the [`joss-reviews`](https://github.com/openjournals/joss-reviews) repository. The first thing you should do is set yourself to 'not watching':
+
+![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)
+
+Please note, that by not watching the reviews repository, you will still receive notifications for issues (reviews) where you are `@mentioned`.
+
+**Curate your GitHub notifications experience**
```

Comment:
Another thing is that if some other editor mentioned all the editors in one review (e.g. asking a particular question) and you think it's not relevant to you, you can manually unsubscribe from that particular review issue.

Reply:
:+1: for this. Also making sure people use the `joss-eics` instead of `joss-editors` (the names are a bit confusing).